Gems/GradientSignal/Code/Source/Components/GradientTransformComponent.cpp

```diff
@@ -276,24 +276,29 @@ namespace GradientSignal
 
     void GradientTransformComponent::Activate()
     {
+        m_dirty = false;
+        m_gradientTransform = GradientTransform();
+
+        // Update our GradientTransform to be configured correctly.  We don't need to notify dependents of the change though.
```

Comment:
Just like the Declaration of Independence!
![image](https://user-images.githubusercontent.com/82224783/147773328-9b812b20-5e86-46dc-9c6f-f5ed5a43abea.png)

But since I'm in here anyways, I'll change it.